internal-functions

```diff
@@ -128,6 +128,14 @@ fn-query-status(){
     AUTH_URL="$AUTH_URL/Auth"
   fi
 
-  local USER_INFO_URL="https://$AUTH_URL/v2/users/$USER/"
-  curl -sk --max-time 3 --header "$HEADER" "$USER_INFO_URL" | jq ".$FIELD" | grep -q 'true'
+  local USER_INFO_URL="https://$AUTH_URL/v2/users/$USER/"  
+  RESPONSE=$(curl -sS --insecure --max-time 10 --retry 5 --header "$HEADER" --write-out "\n%{http_code}" "$USER_INFO_URL")
+  RESPONSE_BODY=$(echo "$RESPONSE" | awk "NR==1")
+  STATUS_CODE=$(echo "$RESPONSE" | awk "NR==2")
```

Comment:
:cow2: I believe we want to be double quoting the vars. here according to our style guide: https://github.com/plotly/dev-docs/blob/master/reference/styleguides/bash.md

Reply:
Good idea. Done!